src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -863,13 +870,23 @@ public static object DeepCopyInner(object original)
             {
                 copy = copier(original);
                 SerializationContext.Current.RecordObject(original, copy);
+                return copy;
             }
-            else
+
+            var externalSerializer = externalSerializers.FirstOrDefault(serializer => serializer.IsSupportedType(t));
```

Comment:
@jason-bragg @galvesribeiro The option for multiple serializers is useful if you just want  to plug in more than one external serializer without writing any code. For example, using the Bond serializer for anything with the Bond SchemaAttribute and then a custom implementation for other types.  In most scenarios, I imagine that this would be the usage pattern.

That said, you can easily do more complex routing by implementing IExternalSerializer as discussed above.  

The benefit of having multiple serializers is the flexibility to do either.